src/main/java/org/janelia/saalfeldlab/n5/readdata/ByteArraySplittableReadData.java

```diff
@@ -0,0 +1,47 @@
+package org.janelia.saalfeldlab.n5.readdata;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+
+class ByteArraySplittableReadData implements ReadData {
```

Comment:
maybe rename to `ByteArrayReadData` since we're not doing the splittable part yet?
I'm also happy to keep the name as is since the splitting will come soon.